client/src/main/java/com/vesoft/nebula/AbstractClient.java

```diff
@@ -33,9 +35,10 @@
      * @param connectionRetry The number of retries when connection failure.
      * @param executionRetry  The number of retries when execution failure.
      */
-    public AbstractClient(List<HostAndPort> addresses, int timeout,
+    public AbstractClient(List<HostAndPort> addresses, int timeout, int connTimeout,
                           int connectionRetry, int executionRetry) {
         checkArgument(timeout > 0);
+        checkArgument(connTimeout >= 0);
```

Comment:
the `connTimeout` could be zero?

Reply:
See the constructor of TSocket, connectionTimeout is initialized with 0 by default

Reply:
Do you mind using `connectionTimeout` ?

Reply:
OK. connectionTimeout looks better